site/guide/configure-model-inventory.qmd

```diff
@@ -0,0 +1,28 @@
+---
+title: "Configure models in the inventory"
+---
+
+Customize the model inventory to fit your needs.
+
+## Prerequisites
+
+Any user can complete these steps.
+
+## Steps
+
+1. [Log in to the {{< var vm_platform >}}]({{< var vm_url >}}).
+
+2. Navigate to the **Model Inventory**.
+
+3. Use the top buttons to organize the model inventory to your preference:
```

Comment:
These are all sorts of different topics, and don't all fall under "Configure models"... I would suggest:

- A `Working with the model inventory` top-level page, like how there are `Working with model documentation`, `Working with documentation templates` etc... 
- Then I'd break this main page into the following sub-sections:
1. Search, filter, and sort models
- I'd just do three sub-sections here, header 3 perhaps (Search models, Filter models, Sort models)
2. Manage inventory layout 
- I'd do the manage column stuff here, then a sub-section under that (h2?): "Toggle display options" or something...
- Then wrap the following sub-pages into this "section":
1. Register models in the inventory
5. Edit model inventory fields